elastic/apm/index-template.json

```diff
@@ -4,6 +4,6 @@
     "_meta": {
         "description": "Placeholder template that'd be used when the x-pack-apm car is not used, e.g. in test mode"
     },
-    "composed_of": ["apm@settings"],
+    "composed_of": ["common-apm@custom"],
```

Comment:
What's the reason for defining another non-standard `@custom` component template? Can we use `traces@custom`, `metrics@custom`, and `logs@custom` here?

Reply:
oops, yes you are right. I should just use the standard custom component.